presto-spi/src/main/java/com/facebook/presto/spi/Connector.java

```diff
@@ -15,6 +15,7 @@
 
 import com.facebook.presto.spi.security.ConnectorAccessControl;
 import com.facebook.presto.spi.session.PropertyMetadata;
+import com.facebook.presto.spi.transaction.IsolationLevel;
 
 import java.util.List;
 import java.util.Set;
```

Comment:
How about making it an `abstract` implementation of `TransactionalConnector` then? It seems weird that we now have two unrelated interfaces that are used for the same thing